README.md

```diff
@@ -1,2 +1,140 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 My Portofolio <a name="about-project"></a>
+
+This project has been created for Big 5. Which is the largest and most influential event for the construction industry with its global hub in Dubai acting as the gateway between East and West.
+
+Our mission is to unite the global construction community and provide leading-edge innovation, knowledge, and business opportunities for industry stakeholders to connect, grow, and succeed.
+
+We believe in accelerating progress in the construction industry to support economic growth and promote a healthier and more sustainable built environment around the world.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML.
+- CSS.
+- JavaScript
+- LINTERS.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Big5</a></li>
+    <li><a href="https://reactjs.org/">Urban Design & Landscape Expo</a></li>
+    <li><a href="https://reactjs.org/"></a>Dubai Expo 2020</li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Added html.index file.
+- Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web Browsers
+- Code Editor.
+- Git -smc.
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command.
+
+### Install
+
+Install this project with:
+Run this command:
+
+- cd my-project
+- npm install
+
+### Usage
+
+To run the project, execute the following command:
+OPen index.html using live server.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Qaisar Abbas**
+
+- My Video Presentation:(https://www.loom.com/share/ff730e65243143dbbacefbd465bc5a13?sid=31c52f21-5ed3-4be5-94d5-3aa4d3739b93)
+- GitHub: [@githubhandle](https://github.com/Kaiserabbas/)
+- Twitter: [@twitterhandle](https://twitter.com/AbbasKayser)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/qaisar-abbas-21a93840/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add Header
+- [ ] Add Navigation Bar
+- [ ] Add Headline
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, Write to me and give me a good rating.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
- [ ] please kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).